### PR TITLE
Lookup releases in "origin" namespace if release contains "okd"

### PR DESCRIPTION
**Pull request by vrutkovs**
This would enable testing OKD releases which may add arbitrary release
edges

/cc @smarterclayton

### CODE REVIEW

File: manager.go [hunk 1]
```diff
@@ -638,23 +638,31 @@ func (m *jobManager) resolveImageOrVersion(imageOrVersion, defaultImageOrVersion
 		return unresolved, "", nil
 	}
 
-	is, err := m.imageClient.ImageV1().ImageStreams("ocp").Get("release", metav1.GetOptions{})
```
Comment (smarterclayton) [inline manager.go:641]: I would suggest instead changing to a for loop where you walk over `[]string{"ocp","origin"}` and search each dir.  If no match, exit.  You might want to extract the code below into a function that takes an image stream and then invoke the method in a loop.

Comment (smarterclayton) [reply manager.go:641]: Build the URL based on the image stream name you find (switch statement), move that into it's own method.

Comment (vrutkovs) [reply manager.go:641]: Updated PTAL. Didn't move this to the function yet so that #81 would not cause too many merge conflicts

**Comment by smarterclayton** [comment]
Gotta rebase

**Pull request merged**